app/src/modules/network/conn_info.cddl

```diff
@@ -0,0 +1,13 @@
+; CDDL schema to encode objects like defined in https://github.com/hello-nrfcloud/proto-map/blob/saga/lwm2m/14203.xml
+; depends on lwm2m_senml definitions
+
+energy-estimate = {
+	bn => "14203/0/",                    ; Connection Information object
+	n => "11",                           ; Energy Estimate resource ID
+	vi => int,                           ; temperature in degrees Celsius
```

Comment:
```suggestion
	vi => 5...9,                           ; Energy Estimate
```

Value has a range.

Reply:
I have used double dots now (5..9) because triple dots exclude the upper bound. Here we want both 5 and 9 to be included as valid values.

Reply:
Ah, i was not aware.